packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/Messages.md

```diff
@@ -97,6 +100,36 @@ You can apply a `clickedAriaLabel` and `clickedTooltipContent` once a button is
 
 ```
 
+### Message feedback
+
+When a user selects a positive or negative [message action](#message-actions), you can display a message feedback card that acknowledges their response and provides space for additional written feedback. These cards can be manually dismissed via the close button and the thank-you card can be [configured to time out automatically](/patternfly-ai/chatbot/messages#message-feedback-with-timeouts).
+<br /><br />
```

Comment:
Just an fyi that we have an issue to fix spacing in our docs (https://github.com/patternfly/patternfly-org/issues/4257), so that we don't have to add these breaks. Not sure when that will get fixed or if that will trickle down to chatbot docs without requiring additional work in this repo, but wanted to mention it because we will need to just make sure to remove these whenever it does get fixed. More of a mental note for myself that anything else 😊